bookstore-jpa/src/main/java/org/books/data/entity/Order.java

```diff
@@ -7,10 +7,11 @@
 import java.util.List;
 
 @NamedQueries({
-        @NamedQuery(name = "Order.findByNumber", query = "from Order o where UPPER(o.number) = UPPER(:number)"),
-        @NamedQuery(name = "Order.findByCustomerAndYear", query = "from Order o where o.customer = :customer AND EXTRACT(YEAR from o.date) = :year")
+    @NamedQuery(name = "Order.findByNumber", query = "from Order o where UPPER(o.number) = UPPER(:number)"),
+    @NamedQuery(name = "Order.findByCustomerAndYear", query = "from Order o where o.customer = :customer AND EXTRACT(YEAR from o.date) = :year")
 })
 @Entity
+@Table(name="command")
```

Comment:
Could we maybe have a expressive table name? Maybe like PURCHASE_ORDER or something similar?